Signals.gd

```diff
@@ -0,0 +1,3 @@
+extends Node
+
+signal player_health_changed(value)
```

Comment:
I'm loving this global singleton stuff, it makes it very simple to share events between nodes without coupling their logic